experimenter/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -22,7 +26,31 @@ def handle(self, *args, **options):
             NimbusFeatureConfig.objects.values_list("application", "slug")
         )
 
-        for feature in Features.all():
+        # A mapping of (application, slug) to feature configs.
```

Comment:
It's application and feature slug?

Reply:
Yeah I'll update this comment to mention which slug.